MessageBroker/ConnectionBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace MessageBroker
+{
+    public static class ConnectionBuilder
+    {
+        public static void AddMessageConsumer(this IServiceCollection services, string queueName , Action<MessagingBuilder> builderFn = null)

```

Comment:
Comment mist voor deze methode, terwijl dit juist een methode is die gebruikt wordt!